src/libs/kata-types/src/capablities.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use bitmask_enum::bitmask;
+
+#[bitmask(u8)]
+enum Bitmask {
```

Comment:
IMO, you should add document comments on these four variables to explain what they mean.

Reply:
Also, the enum name should not be `BItmask`, it should be something like `CapabilitiyBits`

Reply:
Agree. Thanks !